tests/integration/conftest.py

```diff
@@ -20,17 +20,26 @@
 import pytest
 
 from tests.integration.cluster_fixture import TrivupFixture
+from tests.integration.cluster_fixture import ByoFixture
 
 work_dir = os.path.dirname(os.path.realpath(__file__))
 
 
+def create_trivup_cluster():
+    return TrivupFixture({'with_sr': True,
+                          'debug': True,
+                          'cp_version': 'latest',
+                          'broker_conf': ['transaction.state.log.replication.factor=1',
+                                          'transaction.state.log.min.isr=1']})
+
+
+def create_byo_cluster():
+    return ByoFixture({'bootstrap.servers': 'localhost:9092'})
```

Comment:
We should avoid these hardcoded ports, I often run multiple clusters in parallel (for parallel testing).
trivup solves this by allocating dynamic ports, and I think we should base these bootstraps on env BROKERS instead of hardcoded, and error out if BROKERS is not set.
It is much less error prone, much more flexible, and more user friendly, than these hidden fragile assumptions.